modin/pandas/test/test_io.py

```diff
@@ -1259,7 +1287,7 @@ def test_to_parquet(self):
         reason="The reason of tests fail in `cloud` mode is unknown for now - issue #3264",
     )
     def test_read_parquet_2462(self):
-        test_df = pd.DataFrame(
+        test_df = pandas.DataFrame(
```

Comment:
Why is this replaced to pandas usage?

Reply:
We are testing `read_parquet` here, so it follows as a test that we test the read on a Pandas Dataframe because the `read_parquet` method is Modin's.

Reply:
To expand on what @naren-ponder said, the new Modin `to_parquet` does not reproduce the bug that we fixed in #2462. It was incorrect written at first, but not caught because Modin was using pandas `to_parquet`

Reply:
ok, makes sense to me